homeassistant/components/mqtt/__init__.py

```diff
@@ -54,6 +57,7 @@
 DEFAULT_QOS = 0
 DEFAULT_RETAIN = False
 DEFAULT_PROTOCOL = PROTOCOL_311
+DEFAULT_TLS_INSECURE = True
```

Comment:
You're right - I initially labelled the variable tls_secure but later changed it to reflect with variable name paho used but forgot to to change the the value.